nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/test/java/org/nuxeo/ecm/platform/picture/core/test/TestPictureResize.java

```diff
@@ -71,6 +73,7 @@ public void closeOperationContext() {
 
     @Test
     public void testResizer() throws Exception {
+        assumeFalse(SystemUtils.IS_OS_MAC);
```

Comment:
Also please add an inline comment with
```
TODO: NXP-27622
```
so it's visible directly in the code.

Reply:
Arf I put it in the original, then lost it in a bad git manoeuvre...